.github/workflows/release.yaml

```diff
@@ -5,46 +5,50 @@ on:
     - master
   schedule:
   - cron: "0 */8 * * *"
+  workflow_dispatch: {}
```

Comment:
I added this so I could trigger the build over and over while testing various options. Feel free to remove it if you don't use it.